price_pusher/src/injective/injective.ts

```diff
@@ -116,12 +118,14 @@ export class InjectivePricePusher implements IPricePusher {
 
   private async signAndBroadcastMsg(msg: Msgs): Promise<TxResponse> {
     const chainGrpcAuthApi = new ChainGrpcAuthApi(this.grpcEndpoint);
-    const account = await chainGrpcAuthApi.fetchAccount(
+    // Fetch the latest account details only if it's not stored.
+    this.account ??= await chainGrpcAuthApi.fetchAccount(
```

Comment:
wouldn't be better to periodically sync with the rpc on this? (might be helpful if multiple instances of one key are running)

Reply:
we are refetching it if the sequence number is a wrong one. else we won't need to refetch it. 
in that sense it is syncing with the rpc in some way.